convertToSingleHost.js

```diff
@@ -0,0 +1,94 @@
+const fs = require("fs");
+const host = process.argv[2];
+const hosts = ["excel", "onenote", "outlook", "powerpoint", "project", "word"];
+const util = require("util");
+const readFileAsync = util.promisify(fs.readFile);
+const unlinkFileAsync = util.promisify(fs.unlink);
+const writeFileAsync = util.promisify(fs.writeFile);
+
+async function modifyProjectForSingleHost(host) {
+  if (!host) {
+    throw new Error("The host was not provided.");
+  }
+  if (!hosts.includes(host)) {
+    throw new Error(`'${host}' is not a supported host.`);
+  }
+  await convertProjectToSingleHost(host);
+  await updatePackageJsonForSingleHost(host);
+}
+
+async function convertProjectToSingleHost(host) {
+  // copy host-specific manifest over manifest.xml  
+  const manifestContent = await readFileAsync(`./manifest.${host}.xml`, "utf8");
+  await writeFileAsync(`./manifest.xml`, manifestContent);
+
+  // copy host-specific App.tsx over src/taskpane/app/components/App.tsx
+  host = getHostName(host);
```

Comment:
Consider using `const hostname = ` here so you don't redefine host. This is because `host` is still used later and should remain the original lowercase.